app/controllers/cases/data_requests_controller.rb

```diff
@@ -101,8 +104,8 @@ def set_data_request
       @data_request = DataRequest.find(params[:id]).decorate
     end
 
-    def set_commissioning_document
-      @commissioning_document = @data_request.commissioning_document&.decorate
+    def set_data_request_area
+      @data_request_area = DataRequestArea.find(params[:data_request_area_id])
```

Comment:
Can't this be inferred from the data_request?

Reply:
I tried this `@data_request.data_request_area` but It fails when trying to create a new data request with NilClass. Unless there's another way to retrieve it?

Reply:
Your right, before the data request exists you will need to load it like this. 
However it should be loading it via the case otherwise you could get a DataRequestArea that belongs to a different case by changing the URL.